agent/config/types.go

```diff
@@ -218,4 +218,10 @@ type Config struct {
 
 	// TaskMetadataBurstRate specifies the burst rate throttle for the task metadata endpoint
 	TaskMetadataBurstRate int
+
+	// SharedVolumeValidateNameOnly is config option used to short-circuit volume validation against a
+	// provisioned volume, if true. If false, we perform deep comparison including driver options and labels.
+	// For comparing shared volume across 2 instances, this should be set to true as docker does not propagate
+	// driver options and labels through volume drivers.
+	SharedVolumeValidateNameOnly bool
```

Comment:
What is the actual environment variable name?

Reply:
added ECS_SHARED_VOLUME_VALIDATION_NAME_ONLY